clients/drcachesim/simulator/snoop_filter.cpp

```diff
@@ -120,20 +117,23 @@ snoop_filter_t::snoop_eviction(addr_t tag, int id)
     coherence_table_entry_t *coherence_entry = &coherence_table_[tag];
 
     // Check if sharer list is initialized.
-    assert(coherence_entry->sharers.size() == (uint64_t)num_snooped_caches_);
+    assert(!coherence_entry->sharers.empty());
```

Comment:
Would it be useful to also check `coherence_table_.find(tag) != coherence_table_.end()`?